R/create.ccf.summary.heatmap.R

```diff
@@ -118,14 +129,32 @@ create.ccf.summary.heatmap <- function(
         hm.args$text.col <- ifelse(arr > contrast.thres, 'white', 'black');
         }
 
+    if (same.as.matrix) {
+        hm.args$yat <- 1.5
```

Comment:
For parameters added here (line 132 - 148), is everything necessary? Generally, we want to avoid hard coding project/user specific params